openquakeplatform/openquakeplatform/ript/views.py

```diff
@@ -62,6 +71,7 @@ def validate_nrml(request):
     from openquake.commonlib import nrml
 
     xml_text = request.POST.get('xml_text')
+    func_type = request.POST.get('func_type')
```

Comment:
Is it correct that we accept `func_type` to be undefined or not included in a specific set? Otherwise we should check it here instead of line 115, and export nothing in case the check fails.